It happens game's files got corrupted.
Skip downloading pre-made data & pool cache from BYAR http Server.
Not sure about the root cause but looks like using only pr-downloader works.
Otherwise this can be related to mismatch from BYAR & system pr-downloader, only
when versions get out of sync.

While there, also skip downloading engine updates (not built for openbsd).

Fix enginePath to use system bin only.

Index: launcher/src/launcher_wizard.js
--- launcher/src/launcher_wizard.js.orig
+++ launcher/src/launcher_wizard.js
@@ -92,6 +92,7 @@ class Wizard extends EventEmitter {
 				}
 			}
 
+			/*
 			config.downloads.resources.forEach((resource) => {
 				steps.push({
 					name: 'resource',
@@ -113,6 +114,7 @@ class Wizard extends EventEmitter {
 					}
 				});
 			});
+			*/
 
 			if (config.downloads.games && config.downloads.games.length > 0) {
 				steps.push({
@@ -265,15 +267,7 @@ class Wizard extends EventEmitter {
 			}
 		}
 
-		let enginePath;
-		if (config.launch.engine_path != null) {
-			enginePath = config.launch.engine_path;
-		} else {
-			const engineName = config.launch.engine || config.downloads.engines[0];
-			if (engineName != null) {
-				enginePath = path.join(springPlatform.writePath, 'engine', engineName, springPlatform.springBin);
-			}
-		}
+		let enginePath = '/usr/local/bin/spring';
 		if (enginePath != null) {
 			steps.push({
 				name: 'start',
